nbconvert/filters/tests/test_markdown.py

```diff
@@ -45,20 +48,21 @@ class TestMarkdown(TestsBase):
         ('test', 'https://google.com/'),
     ]
 
-
     @dec.onlyif_cmds_exist('pandoc')
     def test_markdown2latex(self):
         """markdown2latex test"""
         for index, test in enumerate(self.tests):
-            self._try_markdown(markdown2latex, test, self.tokens[index])
+            self._try_markdown(partial(convert_pandoc, from_format='markdown',
+                                       to_format='latex'), test,
+                               self.tokens[index])
 
     @dec.onlyif_cmds_exist('pandoc')
     def test_markdown2latex_markup(self):
         """markdown2latex with markup kwarg test"""
         # This string should be passed through unaltered with pandoc's
         # markdown_strict reader
         s = '1) arabic number with parenthesis'
-        self.assertEqual(markdown2latex(s, markup='markdown_strict'), s)
+        self.assertEqual(convert_pandoc(s, 'markdown_strict', 'latex'), s)
```

Comment:
@Carreau  (part 3 of 3) In this case `convert_pandoc` is being invoked with a source argument, but with no named arguments.

This is why I wasn't sure what to include in the warning string.